src/main/scala/io/sdkman/changelogs/MyBatisMigrations.scala

```diff
@@ -0,0 +1,78 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "062")
+class MyBatisMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001-add_mybatis_migrations_candidate",
+    author = "xshyamx"
+  )
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "migrate",
+      name = "Apache MyBatis Migrations",
+      description =
+        "Thus, the MyBatis Schema Migration System (or MyBatis Migrations for short) seeks to provide database migrations for any database (new or existing) and make the current status of the database easily accessible and comprehensible.",
+      websiteUrl = "https://mybatis.org/migrations/",
+      distribution = "UNIVERSAL"
+    ).insert()
+  }
+
+  @ChangeSet(
+    order = "002",
+    id = "002-add_correct_mybatis_migrations_bundle_urls",
+    author = "xshyamx"
+  )
+  def migration002(implicit db: MongoDatabase) = {
+    List(
+      "3.3.9",
+      "3.3.8",
+      "3.3.7",
+      "3.3.6",
+      "3.3.5",
+      "3.3.4",
+      "3.3.3",
+      "3.3.2",
+      "3.3.1",
+      "3.3.0",
+      "3.1.1"
+    ).map(
+        version =>
+          Version(
+            candidate = "migrate",
+            version = version,
+            url =
+              s"https://github.com/mybatis/migrations/releases/download/mybatis-migrations-$version/mybatis-migrations-$version-bundle.zip"
```

Comment:
The releases are not great to depend on as they are removed after some time. We usually depend on the archives URLs.

Reply:
Will change. Initially, I only saw the jar files being published to maven repo but, looking at the version directory the bundle files are also present.

Reply:
Sorry the https://archive.apache.org/dist/ does not contain MyBatis Migrations only the maven repository does